cloud-sql-proxy.yaml

```diff
@@ -0,0 +1,39 @@
+package:
+  name: cloud-sql-proxy
+  version: 2.7.0
+  epoch: 0
+  description: The Cloud SQL Auth Proxy is a utility for ensuring secure connections to your Cloud SQL instances
+  target-architecture:
+    - all
+  copyright:
+    - license: Apache-2.0
+  checks:
+    disabled:
+      - empty
+
+environment:
+  contents:
+    packages:
+      - ca-certificates-bundle
+      - busybox
+      - git
+      - go
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/GoogleCloudPlatform/cloud-sql-proxy
+      tag: v${{package.version}}
+      expected-commit: 2527e5aedbca617579a528070e88a6e02c4d88d1
+
+  - runs: |
+      CGO_ENABLED=0 go build -ldflags "-X github.com/GoogleCloudPlatform/cloud-sql-proxy/v2/cmd.metadataString=container"
```

Comment:
Any reason not to use the `go/build` pipeline here?

Reply:
Also, this is going to output the binary in the workspace dir. IIRC that doesn't remain in the final package.

Reply:
used go/build